Docs 6113 mq ui changes for november 2 release (#46)
mulesoft/docs-mq: modules/ROOT/pages/mq-queues.adoc

. Log in to Anypoint Platform.
. Click *MQ* in the left navigation bar or the main Anypoint Platform screen.
. Click *Destinations*.

> . Click the blue plus icon to display the menu:

Reviewer: ```suggestion
. Click the blue plus icon (+) to display the menu:
```

** *Message TTL* (Time to live) specifies how long unprocessed messages persist before being deleted. 
+
The maximum TTL for a message is two weeks.
** *Default Acknowledgment Timeout* specifies how long a message waits before being put back into the queue in the event of server failure when the message is not acknowledged.
+
The lock makes the message unavailable to other apps while locked, but does not block other messages from being read.
+
The maximum acknowledgement timeout is 12 hours.
** *Assign Default Delivery Delay* specifies how long to delay delivery for messages sent to the queue.
+
You can specify to delay message delivery in seconds, minutes, hours, or days. When you send a message, you can override the default delivery delay. See xref:mq-understanding.adoc#delay_queues[Delay queues].

> ** *Encryption* encrypts all messages in the queue. See xref:mq-understanding.adoc#encrypt-queue[Encrypted queues].

Reviewer: ```suggestion
** *Encryption* encrypts all messages in the queue. 
See xref:mq-understanding.adoc#encrypt-queue[Encrypted queues].
```

. In *Anypoint Platform > MQ*, click *Destinations* in the left pane.
. Click the queue type for the queue to display its details in the right pane.
. Click the *Purge Messages* icon in the details pane:
+
.The arrow shows the *Purge Message* icon in the details pane.
image::mq-purge-msgs-icon.png[Purge Messages icon in the details pane]

> . In the *Purge Messages* confirmation popup, select the checkbox, and then click *Delete Messages* to verify that you want to purge all messages in the queue:

Reviewer: ```suggestion
. In the *Purge Messages* confirmation box, select the checkbox, and click *Delete Messages* to verify that you want to purge all messages in the queue.
```

. Click the queue type for the queue to display its details in the right pane.
. Click the trash can icon in the upper right of the details pane:
+
.The arrow shows the trash can icon in the details pane for deleting a queue.
image::mq-delete-queue-icon.png[Trash can icon in the details pane for deleting a queue]
+

> . In the *Delete Queue* confirmation popup, select the checkbox, and then click *Delete Queue* to verify that you want to delete the queue:

Reviewer: ```suggestion
. In the *Delete Queue* confirmation box, select the checkbox and click *Delete Queue* to verify that you want to delete the queue.
```

* Viewing a message and returning it to the queue counts as a NACK operation and is considered an unsuccessful delivery attempt.
* Deleting the message in the *Message Browser* prevents it from being counted against the maximum deliveries, but the message is lost.

> === How Messages are Sent to a DLQ

Reviewer: ```suggestion
=== How Messages Are Sent to a DLQ
```
Verbs are always capitalized, no matter how short (Is, Are...).  :-)
